spec/registry.md

```diff
@@ -796,6 +796,11 @@ output.
 If both are specified, a _Bad Option_ error MUST be emitted
 and a _fallback value_ used as the resolved value of the _expression_.
 
+If the _operand_ of the _expression_ is an implementation-defined date/time type,
+it can include _style options_, _field options_, or other option values.
```

Comment:
I think we should say `MUST` here, or at least `SHOULD`, and be specific about which options are required to be present (all of them?). The purpose here is be to allow custom function authors to extend MF2 and be able to rely on the implementations exposing these options.

Reply:
I don't think we can `MUST` or `SHOULD` that implementation-defined date/time types contain options. The text here needs to be implementer focused:

```suggestion
Implementations MAY permit callers to supply _style options_, _field options_, or other option values
as part of implementation-defined datetime types or values.
```

I use `MAY` here because `SHOULD` means that every implementation must do so unless they have a good reason not to. It will be difficult to test.